CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
 - Added an `core::error::Error` implementation for `Error` to ease
   integration with error-handling crates. (requires the **unstable** feature)
 - Added partial support for the TCG protocols for TPM devices under `uefi::proto::tcg`.
+- `FileType`, `FileHandle`, `RegularFile`, and `Directory` now implement `Debug`.
```

Comment:
Looks like git rebased this to the wrong place -- it's under 0.19.0 instead of unreleased